src/System.Private.Xml/src/System/Xml/Serialization/Compiler.cs

```diff
@@ -27,9 +27,6 @@ namespace System.Xml.Serialization
 
     internal class Compiler
     {
-#if !XMLSERIALIZERGENERATOR
-        private bool _debugEnabled = DiagnosticsSwitches.KeepTempFiles.Enabled;
-#endif
```

Comment:
We don't set XMLSERIALIZERGENERATOR in any of the System.Private.Xml builds. Some of the Xml code is being referenced by System.Xml.SerializerGenerator and that contract sets this define. In this case, looks like System.Private.Xml is no longer using this bool and it can be removed.